### PR TITLE
Added the bathrobe

### PR DESCRIPTION
**Pull request by atakiya**
## About The Pull Request
Adds a bathrobe.
Ported from virgo.

## Changelog
:cl:
imageadd: Added bathrobe
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
Make it obtainable in game by loadout and some vendor?

**Comment by atakiya** [comment]
> Make it obtainable in game by loadout and some vendor?

Suggestions?

**Comment by atakiya** [comment]
> > Make it obtainable in game by loadout and some vendor?
> 
> Suggestions?

For the vendor that is-

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -199,3 +199,9 @@
 	icon_state = "keyholesweater"
 	can_adjust = FALSE
 
+/obj/item/clothing/under/misc/bathrobe
+	name = "bathrobe"
+	desc = "A fluffy robe to keep you from showing off to the world."
+	icon_state = "bathrobe"
+	worn_state = "bathrobe"
```
Comment (Azarak) [inline code/modules/clothing/under/miscellaneous.dm:206]:
```suggestion
	worn_icon_state = "bathrobe"
```

**Comment by Azarak** [comment]
> > > Make it obtainable in game by loadout and some vendor?
> > 
> > 
> > Suggestions?
> 
> For the vendor that is-

Not sure. Clothesmate would be fine. Perhaps crafting this with cloth aswell.
I imagine that mappers should put those near saunas / pools. Maybe a preset wooden locker with those?

**Comment by atakiya** [comment]
> > > > Make it obtainable in game by loadout and some vendor?
> > > 
> > > 
> > > Suggestions?
> > 
> > 
> > For the vendor that is-
> 
> Not sure. Clothesmate would be fine. Perhaps crafting this with cloth aswell. I imagine that mappers should put those near saunas / pools. Maybe a preset wooden locker with those?

Going to say that will be out of scope of this PR. It's ready as is.
Mappers can add it to pool/sauna areas and such later.

**Review by Azarak — APPROVED** [review]

**Pull request merged**